frontend/src/metabase/palette/hooks/useRegisterShortcut.tsx

```diff
@@ -32,6 +35,14 @@ export const useRegisterShortcut = (
         return {
           ...shortcutDef,
           id,
+          perform: _.compose(
+            () =>
+              trackSimpleEvent({
+                event: "keyboard_shortcut_performed",
+                event_detail: id,
+              }),
```

Comment:
gut check: are we really prepared to get MILLIONS of events?

Reply:
Given that we send events on every pageview... probably? I also [asked an adult](https://metaboat.slack.com/archives/C064EB1UE5P/p1744727144433799?thread_ts=1744207041.826819&cid=C064EB1UE5P) haha